Mathlib/Combinatorics/SimpleGraph/Metric.lean

```diff
@@ -128,6 +128,13 @@ theorem edist_eq_one_iff_adj : G.edist u v = 1 ↔ G.Adj u v := by
     exact w.adj_of_length_eq_one <| Nat.cast_eq_one.mp <| h ▸ hw
   · exact le_antisymm (edist_le h.toWalk) (ENat.one_le_iff_pos.mpr <| edist_pos_of_ne h.ne)
 
+/-- Supergraphs have smaller or equal extended distances to their subgraphs. -/
+theorem edist_le_subgraph_edist {G' : SimpleGraph V} {u v : V} (h : G ≤ G') (hr : G.Reachable u v) :
```

Comment:
This ought to be provable without assuming reachability, right? If u isn't reachable from v in G then the rhs is infinity

Reply:
Thank you, I overlooked that.